docs/data/joy/customization/theme-typography/theme-typography.md

```diff
@@ -127,6 +127,12 @@ extendTheme({
 });
 ```
 
+### Customize the font family
+
+To customize the font family, use the `extendTheme` function:
+
+{{"demo": "CustomFontFamily.js"}}
```

Comment:
Overall, I got confused about the difference between this and the above "Changing the default font" section. Could we be more specific about what's different? Both of them seem to be using the `extendTheme`, which, at a glance, makes me thing it's the same.